virttest/qemu_devices/qcontainer.py

```diff
@@ -196,11 +198,34 @@ def initialize_iothread_manager(self, params, guestcpuinfo,
         """
         iothreads_lst = params.objects("iothreads")
         iothread_scheme = params.get("iothread_scheme")
+        iothread_vqs_mapping = params.get("iothread_vqs_mapping", "")
```

Comment:
global to each(image)

Reply:
I think image_iothread_vqs_mapping is better, because we can set image1_iothread_vqs_mapping = t1:0,1,t2:2,3,4,5,6,7 and image2_iothread_vqs_mapping=t1:0,1,2,3,t2:4,5,6,7